spiceaidocs/docs/components/data-connectors/file.md

```diff
@@ -52,3 +75,39 @@ datasets:
 ```
 
 When the file is modified, the acceleration will be refreshed and will include the latest data.
+
+## Examples
+
+### Absolute path
+
+In this example, `path` is an absolute path to the file on the filesystem.
+
+```yaml
+datasets:
+  - from: file://path/to/customer.parquet
+    name: customer
+    params:
+      file_format: parquet
+```
+
+### Relative path
+
+In this example, the path is relative to the directory where the `spicepod.yaml` is located.
+
+```bash
+├── foo
+│   └── yellow_tripdata_2024-01.parquet
+└── spicepod.yaml
+```
+
+```yaml
+datasets:
+  - from: file:foo/yellow_tripdata_2024-01.parquet
```

Comment:
Might be better to specify the following format as well (if supported)
```
- from: ./file:foo/yellow_tripdata_2024-01.parquet
```

Reply:
I don't think that method is supported